kazoo/protocol/connection.py

```diff
@@ -439,7 +439,7 @@ def _send_request(self, read_timeout, connect_timeout):
         if request.type == Auth.type:
             xid = AUTH_XID
         else:
-            self._xid += 1
+            self._xid = (self._xid % 2147483647) + 1
```

Comment:
Only a few digits in the negative portion are used, such as -1, -2
Normal communication using only positive